doc/source/nmod_poly.rst

```diff
@@ -2294,6 +2294,26 @@ of polynomial multiplication.
     Set `g = \operatorname{tanh}(h) + O(x^n)`.
 
 
+Special polynomials
+--------------------------------------------------------------------------------
+
+.. function:: int _nmod_poly_conway(mp_ptr op, ulong prime, slong deg)
+
+    Sets ``op`` to the coefficients to the Conway polynomial `C_{p, d}`, where
+    `p` is ``prime`` and `d` is ``deg``. This is done by checking against Frank
+    Lübeck's database [Lüb2004]_, which has been compressed in FLINT. Returns
+    `1` in case of success and returns `0` in case of failure.
+
+.. type:: struct _prime_degree_struct
+
+.. function:: struct _prime_degree_struct _nmod_poly_conway_rand(flint_rand_t state)
```

Comment:
I wouldn't introduce a struct for this; I would return one value and pass the other via a pointer.

Reply:
This is how I prefer it. It is also a super duper very minor optimization (at least on x86 with System V ABI). Let me know if you still want me to change it.

Reply:
The problem is just that a user who wants to call this function will have to figure out the struct members, and now there's suddenly a more things that ought to be documented than a single function. Besides, this isn't how most functions in FLINT handle returning a tuple (see ``fmpz_get_d_2exp``, ``n_gcdinv``, etc), so it goes a bit against convention. For a single use, a whole extra datatype doesn't seem worth it. No objections to using such private structs for static methods though.

Reply:
Will fix it!